yarn-project/types/src/logs/get_unencrypted_logs_response.ts

```diff
@@ -0,0 +1,16 @@
+import { ExtendedUnencryptedL2Log } from './extended_unencrypted_l2_log.js';
+
+/**
+ * It provides documentation for the GetsUnencryptedLogsResponse type.
+ */
+export type GetsUnencryptedLogsResponse = {
```

Comment:
Should this be `GetUnencryptedLogsResponse`?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/efcec421ce1f729f5a76a2485a4f725e71eb6daf